os/kernel/Kconfig

```diff
@@ -1109,6 +1109,34 @@ config BINMGR_RECOVERY
 		If any fault occurs in a system, it will either restart binary or reboot the system
 		accroding to the configuration.
 
+config COMPRESSED_BINARY
+	bool "Compressed binary support"
+	default n
+	depends on APP_BINARY_SEPARATION
```

Comment:
Is this working with APP_BINARY_SEPARATION only?

Reply:
Will remove this dependency, don't need it